### ISSUE TITLE
browserify/amd doesn't work when not included with node

### ISSUE THREAD

**Issue by mboudreau** [baumandm/angular-numeraljs#17]
I try to use npm for all my dependencies as I don't really see the point of bower since they do exactly the same thing, except npm does it better.  Either way, I've installed it via npm and added it to my project, but fails on the next reload with this error:

```
Uncaught Error: [$injector:modulerr] Failed to instantiate module pageup.scout-app due to:
Error: [$injector:modulerr] Failed to instantiate module pageup.scout due to:
TypeError: Cannot read property 'defaultFormat' of undefined
    at setDefaultFormat (http://localhost:8080/scout/node_modules/angular-numeraljs/dist/angular-numeraljs.js?t=1453872618015:33:20)
    at http://localhost:8080/scout/src/directives/scout/scout.js?t=1453872618015:21:28
    at Object.invoke (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4523:17)
    at runInvokeQueue (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4429:35)
    at http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4438:11
    at forEach (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:340:20)
    at loadModules (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4419:5)
    at http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4436:40
    at forEach (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:340:20)
    at loadModules (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4419:5)
http://errors.angularjs.org/1.4.8/$injector/modulerr?p0=pageup.scout&p1=Typ…080%2Fscout%2Fvendor%2Fangular%2Fangular.js%3Ft%3D1453872618015%3A4419%3A5)
    at http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:68:12
    at http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4458:15
    at forEach (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:340:20)
    at loadModules (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4419:5)
    at http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4436:40
    at forEach (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:340:20)
    at loadModules (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4419:5)
    at createInjector (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:4344:11)
    at doBootstrap (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:1676:20)
    at bootstrap (http://localhost:8080/scout/vendor/angular/angular.js?t=1453872618015:1697:12)
```

I think the property numeral isn't being set properly in your Browserify wrapper.  I recommend that you use the same javascript with browserify for both bower and npm and not have a separate one for each since you can then guarantee that it would fully work all the time.

**Comment by mboudreau** [comment]
nevermind, apparently I need to include numeraljs into my project separately.  Not sure I agree with this approach.

**Comment by baumandm** [comment]
Hi @mboudreau, I agree that numeraljs should be provided to this library automatically.  Are you using version 1.3.0?  In the last version I switched to a UMD module which should work better than the separate commonjs module (and maybe address this issue).

I updated the browserify example to test this, and it is working without a separate require('numeraljs') in the project.  If you're still having issues, let me know how your project is configured so I can try to fix this.
